src/io/files.js

```diff
@@ -714,6 +714,92 @@ p5.prototype.httpGet = function () {
  * @param  {function}      [errorCallback] function to be executed if
  *                                    there is an error, response is passed
  *                                    in as first argument
+ *
+ * @example
+ * <div>
+ * <code>
+ * // Examples use jsonplaceholder.typicode.com for a Mock Data API
+ *
+ * var url = 'https://jsonplaceholder.typicode.com/posts';
+ * var post = { userId: 1, title: 'p5 Clicked!', body: 'p5.js is way cool.' };
+ *
+ * function setup() {
+ *   createCanvas(800, 800);
+ * }
+ *
+ * function mousePressed() {
+ *
+ *  // Pick new random color values
+ *  var r = random(255);
+ *  var g = random(255);
+ *  var b = random(255);
+ *
+ *  httpPost(url, 'json',
+ *      post,
+ *      function (result) {
+ *        strokeWeight(2);
+ *        stroke(r, g, b);
+ *        fill(r, g, b, 127);
+ *        ellipse(mouseX, mouseY, 200, 200);
+ *        text(result.body, mouseX, mouseY);
+ *      },
+ *      function (error) {
+ *        console.error(error);
+ *      });
+ *
+ *  // you can also omit the datatype and httpPost will guess
+ *  httpPost(url,
+ *      post,
+ *      function (result) {
+ *        strokeWeight(2);
+ *        stroke(r, g, b);
+ *        fill(r, g, b, 127);
+ *        ellipse(mouseX, mouseY, 200, 200);
+ *        text(result.body, mouseX, mouseY);
+ *      },
+ *      function (error) {
+ *        console.error(error);
+ *      });
+ *      }
+ *
+ * </code>
+ * </div>
+ *
+ *
+ * <div><code>
+ *
+ *  var url = 'https://invalidURL'; // A bad URL that will cause errors
+ *  var post = { title: 'p5 Clicked!', body: 'p5.js is way cool.' };
+ *
+ *  function setup() {
+ *     createCanvas(800, 800);
+ *  }
+ *
+ *  function mousePressed() {
```

Comment:
Should I add a note about p5.dom ?

Reply:
This form of `mousePressed` is bundled with p5.js, only `p5.Element.mousePressed` is bundled with p5.dom.js so the user probably won't need p5.dom.js when running this example on their own.

Reply:
Cool - new here. Thanks!